samples/cellular/nrf_cloud_multi_service/prj.conf

```diff
@@ -136,3 +136,14 @@ CONFIG_NRF_CLOUD_SEND_DEVICE_STATUS=y
 CONFIG_NRF_CLOUD_SEND_DEVICE_STATUS_NETWORK=y
 CONFIG_NRF_CLOUD_SEND_DEVICE_STATUS_SIM=y
 CONFIG_NRF_CLOUD_SEND_SERVICE_INFO_FOTA=y
+
+CONFIG_MEMFAULT=n
+#CONFIG_MEMFAULT_NCS_PROJECT_KEY=""
+#CONFIG_MEMFAULT_NCS_DEVICE_INFO_CUSTOM=y
+#CONFIG_MEMFAULT_HTTP_PERIODIC_UPLOAD_INTERVAL_SECS=90
+#CONFIG_MEMFAULT_METRICS_HEARTBEAT_INTERVAL_SECS=90
+#CONFIG_MEMFAULT_METRICS_THREADS=n
+
+CONFIG_DOWNLOADER_TRANSPORT_COAP=n
+CONFIG_NRF_CLOUD_COAP_DOWNLOADS=n
+CONFIG_NRF_CLOUD_HTTPS_DOWNLOADS=y
```

Comment:
The CoAP Connector (proxy) has a white-list of domains it allows proxying to, so it will reject proxying to ota-cdn.memfault.com. Disable CoAP for image downloads, so it's downloaded over HTTPS instead.